sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -328,7 +328,7 @@ public void testTextLoad() {
 
   @Test
   public void testCountMinSketch() {
-    Dataset df = context.range(1000);
+    Dataset<Object> df = context.range(1000);
```

Comment:
Hi @rxin If `Dataset<Long>` is used then it seems it will have an issue with the compiler:
```
[error] ...spark/sql/JavaDataFrameSuite.java:331: error: incompatible types:
Dataset<Object> cannot be converted to Dataset<Long>
[error]     Dataset<Long> df = context.range(1000);
```
I will use `Dataset<?>` for now.